# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/zenity/no-man-page.patch
# Copyright (C) 2023 - 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

Manpage fails to generate so remove that step for now.

  - NoTag <notag@maneto.fr>

--- zenity-4.0.1/data/meson.build.vanilla	2024-06-19 23:11:59.587450780 +0200
+++ zenity-4.0.1/data/meson.build	2024-06-19 23:12:10.462450508 +0200
@@ -21,26 +21,3 @@
     args: [desktop_file]
   )
 endif
-
-# manpage -- generated from help2man
-
-help2man = find_program('help2man', required: get_option('manpage'))
-# help2man has to run zenity to generate the man page, which it can't do if
-# we're cross building. We also can't tell help2man what exe wrapper to use.
-if help2man.found() and not meson.is_cross_build() and get_option('manpage')
-  help2man_opts = [
-    '--no-info',
-    '--section=1',
-    '--help-option=--help-all',
-    '--include=@INPUT@',
-  ]
-
-  custom_target('zenity.1',
-                output: 'zenity.1',
-                input: 'zenity.roff',
-                command: [
-                  help2man, help2man_opts, '--output=@OUTPUT@', zenity
-                ],
-                install: true,
-                install_dir: join_paths(zenity_mandir, 'man1'))
-endif
--- zenity-4.0.1/meson.build.vanilla	2024-06-19 23:14:57.610446338 +0200
+++ zenity-4.0.1/meson.build	2024-06-19 23:15:03.436446193 +0200
@@ -105,7 +105,6 @@
 subdir('data')
 subdir('icons')
 subdir('po')
-subdir('help')
 
 gnome.post_install(
   gtk_update_icon_cache: true,
